web/src/click_handlers.js

```diff
@@ -809,6 +809,9 @@ export function initialize() {
     $("body").on("click", ".language_selection_widget button", (e) => {
         e.preventDefault();
         e.stopPropagation();
+        // We need to trigger `blur` to make the tooltip go away after
+        // the click event.
+        $(e.currentTarget).trigger("blur");
```

Comment:
At other places, this is fixed by setting the trigger for tooltip to `mouseenter`. 

Like we have `tippy-delayed-zulip-tooltip` class which does this but also adds a delay while showing the tooltip. So, this can be fixed adding code in `tippy.js` in a similar way.

Reply:
Thanks for the suggestion, I've set the trigger to mouseenter.